subsys/bluetooth/shell/gatt.c

```diff
@@ -1056,6 +1056,16 @@ static int cmd_set(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
+int cmd_att_mtu(const struct shell *shell, size_t argc, char *argv[])
+{
+	uint16_t mtu;
+
+	mtu = bt_gatt_get_mtu(default_conn);
```

Comment:
Sorry for doing this one comment at a time :P

I think we should check if default_conn is connected, since the API will not check for NULL pointer.

```
	if (!default_conn) {
		shell_print(shell, "Not connected");
		return -ENOEXEC;
	}
```

Reply:
Fixed